### ISSUE TITLE
[Android] Cannot read property 'invocationEventNone' of undefined

### ISSUE THREAD

**Issue by cxrod** [Instabug/Instabug-React-Native#387]
### Steps to Reproduce the Problem
Follow migration guide to 8.6.3, then build and run application on android.

### Expected Behavior
Successfully run the app on android

### Actual Behavior
Its working fine on iOS but on android Im getting this error.

![image](https://user-images.githubusercontent.com/16138018/64490774-eb582480-d236-11e9-84dd-161ae241e1d6.png)

### Instabug integration code
```
export const initErrorReporter = () => {
  //Instabug.startWithToken(RNConfig.INSTABUG_API_KEY, [Instabug.invocationEvent.none]);
  Instabug.start(RNConfig.INSTABUG_API_KEY, [Instabug.invocationEvent.none]);
  
  CrashReporting.setEnabled(true);

  // Set color, only for iOS. Go to MainApplication.java to set in Android
  // RN's processColor function is broken, that's why the color is hardcoded
  Instabug.setPrimaryColor(1546610175);

  // Disable welcome message, or show in beta or live
  Instabug.setWelcomeMessageMode(Instabug.welcomeMessageMode.disabled)

  // Disable the Bugs & Feedback. If disabled, both "Report a problem" & "Suggest an improvement" are removed from the Instabug's prompt, and manually showing the bug reporting or feedback doesn't have an effect.
  BugReporting.setEnabled(true);
  
  // Specify which of the feedback or bug options appear in the prompt options
  BugReporting.setReportTypes([BugReporting.reportType.bug, BugReporting.reportType.feedback]);
  
  // Disable the Chats. If disabled, The "Ask a question" (this is the new content instead of "Talk to us") option is removed from Instabug's prompt, the compose button in the chats list is removed, and manually showing the chats compose view of the chats doesn't have an effect.
  Chats.setEnabled(true);
  
  // Disable the Replies. If disabled, the chats list button is removed from Instabug's prompt, the in-app notifications are disabled, and manually showing the chats list doesn't have an effect. 
  Replies.setEnabled(true);
  
  Instabug.setReproStepsMode(Instabug.reproStepsMode.enabledWithNoScreenshots);

  configLocale();
}
```

### SDK Version
Upgrade to 8.6.3 from 8.6.1

### React Native, iOS and Android Versions
RN 0.59.10
Android 9.0

### Device Model
Emulator Nexus One API 28 - Android 9 (Google APIs) - x86

**Comment by alyezz** [comment]
@cxrod This happens when the sdk isn’t linked properly. Make sure you run react-native link instabug-reactnative. If you're still getting this error then something went wrong while linking, you will need to follow the manual linking steps [here] (https://docs.instabug.com/docs/react-native-integration#section-manual-linking)

**Comment by mattetre** [comment]
@alyezz  What is the proper way to initialise with `8.6.3`? The manual instructions you sent still have the `RNInstabugReactnativePackage.Builder` in the `getPackages` method but the upgrade steps say to remove the builder (https://github.com/Instabug/Instabug-React-Native/blob/master/README.md#updating-to-version-863)?

**Comment by lucidtheory** [comment]
agreed, most upgraded rN packages for rN v .60+ utilize the autolinkng. 

The sample project seems to be utilizing autolinking even though it still imports instabug in the `MainApplication.java`

**Comment by alyezz** [comment]
@mattetre Sorry about that, the proper way is to remove the builder, following this link https://github.com/Instabug/Instabug-React-Native/blob/master/README.md#updating-to-version-863
 I will update the docs right away

**Comment by mattetre** [comment]
@alyezz I have removed the builder, but now I am receiving the `'invocationEventNone' of undefined` error

**Comment by mattetre** [comment]
Also when running `react-native link instabug-reactnative` it seems to auto add in the builder to the `MainApplication.java`

**Comment by rollsroyc3** [comment]
Also running into this same issue, upgraded to 8.6.2 last week with no issues. But then saw this and liked the idea of cleaning up my build process, but now running into the same invocationEventNone error

**Comment by alyezz** [comment]
We are aware of this issue and working on a hotfix

**Comment by HeshamMegid** [comment]
Unfortunately, we’ve found an issue in v8.6.3 related to initializing the SDK from Javascript that slipped our testing. We just released v8.6.4, rolling back the changes related to initialization that were included in v8.6.3. This means you’ll have to add the Builder snippet back to your Java code.

We know how inconvenient this is, so please accept our apologies for that. We’re currently working on a fix that would let us bring those initialization changes back.
